wp-content/themes/pub/wporg-learn-2020/css/components/_notice.scss

```diff
@@ -0,0 +1,48 @@
+.notice {
```

Comment:
Most of this came from `_locale_notice.scss`, only the survey notice style is new